README.md

```diff
@@ -1,24 +1,198 @@
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage ](#usage-)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Ruby version
+# Doctors-appointment-backend <a name="hello-rails-react"></a>
 
-* System dependencies
+This web application is the API that serves as the backend to the [ Doctors-appointment-frontend](https://github.com/dialite/Doctors-appointment-frontend)
+Also, it's used for booking doctor appointments.
 
-* Configuration
+## Front-end Repository
+| Repository | Pull request |
+|------------|------------|
+|[Repo link >>](https://github.com/dialite/Doctors-appointment-frontend)|[Pull request Here >>](https://github.com/dialite/Doctors-appointment-frontend/pull/10)|
 
-* Database creation
+## Team members: @dialite, @Aduda-Boaz, @storres20
 
-* Database initialization
+## Backend API routes
+| Route | URL |
+|------------|------------|
+|/users|https://doctor-t64q.onrender.com/api/v1/users|
+|/doctors|https://doctor-t64q.onrender.com/api/v1/doctors|
+|/appointments|https://doctor-t64q.onrender.com/api/v1/appointments|
 
-* How to run the test suite
+![backapi](https://user-images.githubusercontent.com/81504385/231864037-c68e528f-67da-4783-b313-f0e243fd5232.jpeg)
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+## Frontend
+| URL |
+|------------|
+|https://doctorsfinal.netlify.app/|
 
-* ...
+## Kanban Board
+| URL |
+|------------|
+|https://github.com/users/dialite/projects/10/views/1|
+
+![Captura de pantalla de 2023-04-13 14-31-58](https://user-images.githubusercontent.com/81504385/231864497-891f7ada-b648-4482-a7a7-177da8910313.png)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">ROR</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- API that sends information about reservations, appointments, book new appointments
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby on Rails](https://rubyonrails.org/) installed and running. To get more information, read the [installation guide](https://guides.rubyonrails.org/).
+
+- [PostgreSQL](https://www.postgresql.org/) installed and running
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/dialite/Doctors-appointment-backend.git
+```
+
+### Install
```

Comment:
[ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍